sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/catalog/SessionCatalog.scala

```diff
@@ -81,14 +90,27 @@ class SessionCatalog(
     if (conf.caseSensitiveAnalysis) name else name.toLowerCase
   }
 
+  private[sql] def makeQualifiedPath(path: String): Path = {
```

Comment:
it seems like this is used only in tests. Can we remove this and just hard code the qualified prefix in test cases?